examples/joblib_.py

```diff
@@ -18,7 +18,8 @@ def _test_patcher(record):
 
     logger.configure(patcher=_test_patcher)
 
-    funcs = [delayed_with_logger(worker_func, logger)(x) for x in range(3)]
+    worker_func = propagate_logger(worker_func, logger)
+    funcs = [delayed(worker_func)(x) for x in range(3)]
```

Comment:
It may be technically difficult, but it would be ideal if we could output a log in the child process using the loguru configuration of the parent process, without using `loguru_enqueue_and_listen` and `propagate_logger`. If this could be done, it would lower the barriers to using `loguru-parallel`, since it would be possible to use it without modifying existing code that uses `loguru` and `joblib`.